README.md

```diff
@@ -1,2 +1,197 @@
 # planship-nuxt
-Planship Nuxt module
+
+Welcome to `@planship/nuxt`, a [Nuxt module](https://nuxt.com/modules) that makes [Planship](https://planship.io) intergration with [Nuxt](https://nuxt.com) apps a breeze. This module is based on the [`@planship/vue`](https://github.com/planship/planship-vue) plugin and [`@planship/fetch`](https://github.com/planship/planship-js/tree/master/packages/fetch) library.
+
+A complete, working example of a Nuxt app that used the `@planship/nuxt` module can be found at https://github.com/planship/planship-nuxt-demo
+
+# Getting started
+
+Install `@planship/nuxt` with npm, yarn or pnpm:
+
+```sh
+npm install @planship/nuxt
+# or
+yarn add @planship/nuxt
+# or
+pnpm add @planship/nuxt
+```
+
+Then, add `@planship/nuxt` to the `modules` section of your `nuxt.config.ts`:
+
+```ts
+export default defineNuxtConfig({
+  modules: ['@planship/nuxt'],
+})
+```
+
+Finally, configure your Planship product slug and authentication credentials inside `defineNuxtConfig` or via the environmental variables.
+
+```ts
+export default defineNuxtConfig({
+  // ...
+  planship: {
+    productSlug: '<YOUR_PLANSHIP_PRODUCT_SLUG',
+    clientId: '<YOUR_PLANSHIP_API_CLIENT_ID>',
+    clientSecret: '<YOUR_PLANSHIP_API_CLIENT_SECRET>',
+  }
+}
+```
+
+### Configurtation options
+
+***`productSlug`***
+
+Your [Planship product slug](https://docs.planship.io/concepts/products/). The slug can be also defined via the `PLANSHIP_PRODUCT_SLUG` variable.
+
+***`clientId`*** and ***`clientSecret`***
+
+Your Planship [authentication credentials](https://docs.planship.io/integration/#authentication-and-security). They can be also defined via the `PLANSHIP_API_CLIENT_ID` and `PLANSHIP_API_CLIENT_SECRET` variables.
+
+# Usage
+
+## Composables
+
+The `@planship/nuxt` module exports two composables implemented by the `@planship/vue` plugin: `usePlanshipCustomer` and `usePlanship`. The `@planship/vue` plugin is universal-mode friendly, meaning that these composables can be used for both server and client side rendering.
```

Comment:
client side -> client-side

Reply:
This section got replaced by more detailed one from `@planship/vue` README, but I fixed the same type in the new copy.